src/groupeddataframe/splitapplycombine.jl

```diff
@@ -168,63 +168,48 @@ function groupby(df::AbstractDataFrame, cols;
     return gd
 end
 
+const F_TYPE_RULES =
 """
```

Comment:
OK. I will rename them.
But what indentation level would you expect here (please best paste some short example as I was not clear how to properly indent it indeed)?
I did not do indentation because docstrings normally are not indented

Reply:
Just one level (4 spaces)? The idea is that it's the continuation of the first line, so general rules apply.

Reply:
added